docker/conf/docker-sync.yml.dist

```diff
@@ -1,6 +1,7 @@
 version: '2'
 syncs:
     shopsys-framework-sync:
+        sync_userid: 501
```

Comment:
I would consider naming the file `docker-sync-macos.yml.dist` so it's immediately apparent it's for the MacOS.

Reply:
I dont understand why. While we named our docker-compose for linux, just `docker-compose.yml.dist` why is it so important to know that `docker-sync` is for macs.